README.md

```diff
@@ -124,22 +120,20 @@ import com.microsoft.playwright.*;
 
 public class EvaluateInBrowserContext {
   public static void main(String[] args) throws Exception {
-    Playwright playwright = Playwright.create();
-    BrowserType browserType = playwright.firefox();
-    Browser browser = browserType.launch(new BrowserType.LaunchOptions().withHeadless(false));
-    BrowserContext context = browser.newContext();
-    Page page = context.newPage();
-    page.navigate("https://www.example.com/");
-    Object dimensions = page.evaluate("() => {\n" +
-        "  return {\n" +
-        "      width: document.documentElement.clientWidth,\n" +
-        "      height: document.documentElement.clientHeight,\n" +
-        "      deviceScaleFactor: window.devicePixelRatio\n" +
-        "  }\n" +
-        "}");
-    System.out.println(dimensions);
-    browser.close();
-    playwright.close();
+    try (Playwright playwright = Playwright.create();
+         Browser browser = playwright.firefox().launch(new BrowserType.LaunchOptions().withHeadless(false));
+         BrowserContext context = browser.newContext();
+         Page page = context.newPage()) {
+      page.navigate("https://www.example.com/");
+      Object dimensions = page.evaluate("() => {\n" +
+                                          "  return {\n" +
```

Comment:
nit: wrong alignment, let's keep it closer to the left for better readability of the readme file.